test/unit-tests/function/arithmetic/subtractScalar.test.js

```diff
@@ -0,0 +1,267 @@
+// test subtract
+import assert from 'assert'
+
+import approx from '../../../../tools/approx.js'
+import math from '../../../../src/defaultInstance.js'
+import Decimal from 'decimal.js'
+const { subtract, BigNumber } = math
```

Comment:
I think the tests should actually use `subtractScalar` and not `subtract`, and there are no tests needed for matrices here since `subtractScalar` does not support those.